packages/tailwindcss/src/css-functions.ts

```diff
@@ -17,7 +17,7 @@ export function substituteFunctions(ast: AstNode[], pluginApi: PluginAPI) {
     if (node.kind === 'rule') {
       if (
         node.selector[0] === '@' &&
-        node.selector.startsWith('@media ') &&
+        (node.selector.startsWith('@media ') || node.selector.startsWith('@custom-media ')) &&
```

Comment:
we should probably add `@container` to this list too which makes me think… should we just do this for all at-rules?

Reply:
Yeah probably so, technically could use it in `@supports` too although a bit contrived.

Reply:
🫡